src/dynamodb-container.ts

```diff
@@ -194,17 +275,22 @@ export interface TableInitStructure {
 export interface DynamoDBContainerOptions {
   /** Should container be used */
   reuse?: boolean;
+
+  /** Reset data */
+  reset?: boolean;
 }
 
 export class DynamoDBContainer extends GenericContainer {
   private static readonly IMAGE_NAME = 'amazon/dynamodb-local';
   public static readonly MAPPED_PORT = 8000;
+  private shouldResetData: boolean;
 
   constructor(
     private readonly initStructure: Array<TableInitStructure> = [],
-    { reuse }: DynamoDBContainerOptions = {}
+    { reuse, reset = true }: DynamoDBContainerOptions = {}
```

Comment:
Reset should be true?

Reply:
Yes! The current behaviour is for reset to always happen, adding a default of true means it shouldn't break existing implementations but allows a new implementation to opt in to the new behaviour